src/tendencies/advection.jl

```diff
@@ -98,8 +98,10 @@ function explicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx)
     )
     @. Yₜ.c.uₕ[colidx] -=
         ᶜinterp(ᶠω¹²[colidx] × ᶠu³[colidx]) +
-        (ᶜf[colidx] + ᶜω³[colidx]) ×
-        (Geometry.project(Geometry.Contravariant12Axis(), ᶜuvw[colidx]))
+        (
+            Geometry.project(Geometry.Contravariant3Axis(), ᶜf[colidx]) +
```

Comment:
@szy21 It seems this choice to store \^c f as covariant and project to contravariant modifies the mse tables in some cases with no surface elevation.

Reply:
cc @Zhengyu-Huang

Reply:
Could you trigger a longrun build then to make sure the current simulations still work? @Zhengyu-Huang Could you double check this change?

Reply:
> Could you trigger a longrun build then to make sure the current simulations still work? @Zhengyu-Huang Could you double check this change?

Build 758 has been triggered towards answering this.

Reply:
@simonbyrne 

We have a question about some operators:

Are
```
ᶜf = @.Geometry.Covariant3Vector(Geometry.WVector(ᶜf))
Geometry.project(Geometry.Contravariant3Axis(), ᶜf[colidx])
```
and

```
ᶜf = @.Geometry.Contravariant3Vector(Geometry.WVector(ᶜf))
```
the same ?

Reply:
This change has been reverted.